web/template/layout/address-summary-row.gohtml

```diff
@@ -10,7 +10,7 @@
                 </a>
             {{ end }}
         </dd>
-        {{ if and .Address .CanChange }}
+        {{ if and ( not (eq .Address.Line1 "")) .CanChange }}
```

Comment:
you can do `ne ...` instead of `not (eq ...)`

Reply:
Much neater